### PR TITLE
Contents update

### PR DESCRIPTION
**Pull request by jdetle**
Todo: 
- [x] update docs
- [x] finish tests

### CODE REVIEW

**Comment by jdetle** [comment]
I think this is also ready for merge.

File: src/contents.js [hunk 1]
```diff
@@ -51,13 +72,33 @@ export function get(serverConfig, path, params) {
...
+}
+
+/**
  * Creates an AjaxObservable for creating content
  *
  * @param {Object}  serverConfig  - The server configuration
  * @param {string}  path  - The path to the content
  * @param {Object}  model - ^^^^ TODO Above
  *
- * @return  {AjaxObserbable}  An Observable with the request response
```
Comment (rgbkrk) [inline src/contents.js]: Obserbable :wink:

File: src/contents.js [hunk 2]
```diff
@@ -51,13 +72,33 @@ export function get(serverConfig, path, params) {
...
 
 /**
+ * Creates an AjaxObservable for renaming a file.
+ *
+ * @param {Object}  serverConfig  - The server configuration
+ * @param  {string} path - The content to rename.
+ * @param  {Object} model -  ^^TODO
+ * @return  {AjaxObservable}  An Observable with the request response
+ */
+export function rename(serverConfig, path, model) {
```
Comment (rgbkrk) [inline src/contents.js]: What was our thought on rename vs. update here? There was more that could be done with a `PATCH` right?

Comment (jdetle) [inline src/contents.js]: It looks like no, you can only provide a new path in the model.

File: src/contents.js (hunk 1 above)
Comment (jdetle) [inline src/contents.js]: Its Babel written in RxJS

File: src/contents.js (hunk 2 above)
Comment (jdetle) [inline src/contents.js]: Seems like a plan, will change to update.

File: src/contents.js (hunk 1 above)
Comment (rgbkrk) [inline src/contents.js]: Haha, love it.

**Pull request merged**